src-self-hosted/link/MachO.zig

```diff
@@ -143,7 +192,10 @@ pub fn flush(self: *MachO, module: *Module) !void {
     }
 }
 
-pub fn deinit(self: *MachO) void {}
+pub fn deinit(self: *MachO) void {
+    self.segment_cmds.deinit(self.base.allocator);
+    self.@"sections".deinit(self.base.allocator);
```

Comment:
what's the idea behind @"" here?

```suggestion
    self.sections.deinit(self.base.allocator);
```

Reply:
Yeah, same story as with "@"section_64"`, thanks for spotting that one!